libs/math/tests/math/all_tests/basic_math/standard_functions_test.cpp

```diff
@@ -24,17 +24,55 @@
 using namespace fetch::math;
 
 template <typename T>
-class ClampTest : public ::testing::Test
+class StandardFunctionTests : public ::testing::Test
 {
 };
 
 using MyTypes = ::testing::Types<fetch::math::Tensor<float>, fetch::math::Tensor<double>,
                                  fetch::math::Tensor<fetch::fixed_point::FixedPoint<16, 16>>,
                                  fetch::math::Tensor<fetch::fixed_point::FixedPoint<32, 32>>>;
 
-TYPED_TEST_CASE(ClampTest, MyTypes);
+TYPED_TEST_CASE(StandardFunctionTests, MyTypes);
 
-TYPED_TEST(ClampTest, clamp_array_1D_test)
+template <typename TensorType>
+void RandomAssign(TensorType &tensor)
```

Comment:
there actually exist already some helper functions in tensor, such as:

Tensor<T, C>::UniformRandom
Tensor<T, C>::UniformRandomIntegers
Tensor<T, C>::FillUniformRandom
Tensor<T, C>::FillUniformRandomIntegers

At some point we should probably move a lot of this implementation out of tensor into free functions in the math library; but in any case the implementations are already there

Reply:
this was dragged in from elsewhere during a merge, not sure what I should do here to resolve this.